rules/python3.mk

```diff
@@ -0,0 +1,38 @@
+# python3.5.2+
```

Comment:
'+' indicates any later version

---
In reply to: [99458337](https://github.com/Azure/sonic-buildimage/pull/245#discussion_r99458337) [](ancestors = 99458337)